app/views/referee_interface/reference/confirmation.html.erb

```diff
@@ -1,19 +1,27 @@
-<% content_for :title, t('page_titles.give_a_reference.confirmation') %>
-<div class="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-8">
-  <h1 class="govuk-panel__title">
-    Your reference for <%= @reference.application_form.full_name %> has been&nbsp;submitted
-  </h1>
-</div>
+<% if @reference.feedback_status == 'feedback_refused' %>
```

Comment:
It would be nice to be able to say `if @reference.feedback_refused?` but that might require some wrangling of the enum (see bottom of page here: https://api.rubyonrails.org/v5.2.4.1/classes/ActiveRecord/Enum.html).

Reply:
`feedback_status_feedback_refused?` should work, thanks for reminding me of this feature! I think it's preferable to the `==`

Reply:
Oh, it's even simpler, `feedback_refused?` works OOTB!